package.json

```diff
@@ -37,6 +40,7 @@
     "temp": "^0.8.3"
   },
   "scripts": {
+    "download": "node utils/download_data.js",
     "import": "node import.js",
```

Comment:
This wasn't a change in this PR but I noticed an inconsistency: The README recommends  `node import.js`, while all the other importers call `npm start` to import. 

The only difference between `npm start` and `npm import` here is `--max_old_space_size=4096`in `npm start`